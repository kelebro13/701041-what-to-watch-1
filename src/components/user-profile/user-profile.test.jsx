
import renderer from "react-test-renderer";
import {MemoryRouter} from 'react-router-dom';
import UserProfile from "./user-profile";

it(`render correctly UserBlock component`, () => {
  const wrapper = renderer
    .create(<MemoryRouter><UserProfile/></MemoryRouter>)
    .toJSON();

  expect(wrapper).toMatchSnapshot();
});
