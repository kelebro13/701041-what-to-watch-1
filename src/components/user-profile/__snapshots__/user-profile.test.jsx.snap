// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly UserBlock component 1`] = `
<div
  className="user-block"
>
  <a
    className="user-block__link"
    href="/login"
    onClick={[Function]}
  >
    Sign in
  </a>
</div>
`;
